services/loki.nix

```diff
@@ -40,10 +40,10 @@ in {
 
       storage_config = {
         boltdb = {
-          directory = "/var/loki/index";
+          directory = "${dataDir}/index";
```

Comment:
Looking above, I'd rather the dataDir is `/var/db/loki`. That's the standard path for all dbs in NixOS

Reply:
updated `dataDir`